.github/workflows/check-vulnerability-whitesource.yml

```diff
@@ -2,7 +2,7 @@ name: WhiteSource Vulnerability Scan
 
 on:
   schedule:
-    - cron: '0 */2 * * *'
+    - cron: '30 10 * * *'
```

Comment:
Isn't once a day is `0 0 * * *` https://crontab.guru/once-a-day

Reply:
It shouldn't matter. `0 0 * * *` is to initiate the scan at midnight. `30 10 * * *` is to start at 10:30 am everyday. What we used to have is at 10:30 everyday.

Reply:
Ahh I see. forgot about the time part. Thanks!